app/views/step_by_step_pages/show/_required_actions.html.erb

```diff
@@ -46,4 +46,8 @@
   <%= render "govuk_publishing_components/components/inset_text" do %>
     <p class="govuk-body">Not yet claimed for 2i</p>
   <% end %>
+<% elsif @step_by_step_page.status.in_review? %>
+  <%= render "govuk_publishing_components/components/inset_text" do %>
+    <p class="govuk-body">The 2i reviewer is <%= @step_by_step_page.reviewer.name %></p>
```

Comment:
Have we agreed on copy for this? I'd think it should be more status driven, such as "This step by step is currently being 2i reviewed by <name>".

Also would we want to display different copy for the reviewer? "You are reviewing this step by step."...?

Reply:
The current copy is taken from the existing design. Happy to update if we decide otherwise.